modules/rosa-imds-machine-pools-ui.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa_nodes/rosa-managing-worker-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="rosa-imds-machine-pools-ui_{context}"]
+= Enabling Instance Metadata Service on machine pools in {cluster-manager}
+
+You can select your Instance Metadata Service (IMDS) type when creating your cluster in {cluster-manager}. You may select to use both IMDSv1 and IMDSv2, or you may specify to only use IMDSv2.
+
+.Prerequisites
+
+* You installed and configured the latest AWS (`aws`), ROSA (`rosa`), and OpenShift (`oc`) CLIs on your workstation.
+* You logged in to your Red Hat account by using the `rosa` CLI.
+* You have the permissions to create and manage clusters.
+* You have access to {cluster-manager-url}.
+
+.Procedure
+
+. Log in to the web console.
+. Create a ROSA cluster using your preferences.
+. On the *Default machine pool* page, under the *Instance Metadata Service (IMDS)* section, select whether your machine pools use either both IMDSv1 and IMDSv2 or if you only want to use IMDSv2.
```

Comment:
This is slightly ambiguous - the `Default machine pool` is part of the previous step, it happens during the step `Create a ROSA cluster...`, not after as one could understand.

Reply:
@mareklibra The title "Default Machine Pool" is the largest header on this step page. It's how I've seen us refer to this page in [other parts of the documentation](https://docs.openshift.com/rosa/rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-with-customizations.html#rosa-sts-creating-cluster-customizations-ocm_rosa-sts-creating-a-cluster-with-customizations) (see step 6). If you'd like, I can call this "On the **Default machine pool** page of the **Cluster settings** step."

![image](https://github.com/openshift/openshift-docs/assets/16167833/31b38625-83c1-42bc-b340-1628775acfd1)

Reply:
I updated the text to say:

> . In the **Create a ROSA Cluster** wizard on the **Cluster settings** -> **Machine pool** page, under the *Instance Metadata Service (IMDS)* section, select whether your machine pools use either both IMDSv1 and IMDSv2 or if you only want to use IMDSv2.